cache/lru_cache.cc

```diff
@@ -214,11 +220,23 @@ double LRUCacheShard::GetHighPriPoolRatio() {
   return high_pri_pool_ratio_;
 }
 
+double LRUCacheShard::GetLowPriPoolRatio() {
+  DMutexLock l(mutex_);
+  return low_pri_pool_ratio_;
+}
+
 void LRUCacheShard::LRU_Remove(LRUHandle* e) {
   assert(e->next != nullptr);
   assert(e->prev != nullptr);
   if (lru_low_pri_ == e) {
-    lru_low_pri_ = e->prev;
+    if (e->prev != lru_bottom_pri_) {
+      lru_low_pri_ = e->prev;
+    } else {
+      lru_low_pri_ = &lru_;
+    }
+  }
+  if (lru_bottom_pri_ == e) {
```

Comment:
`e` cannot be equal to both `lru_low_pri_` and `lru_bottom_pri_` at the same time, right? If so, this could be `else if`

Reply:
> `e` cannot be equal to both `lru_low_pri_` and `lru_bottom_pri_` at the same time, right? If so, this could be `else if`

I actually might be wrong about this, see above